keps/sig-architecture/1618-conformance-profiles/README.md

```diff
@@ -267,10 +272,36 @@ deploy.
    profiles, profiles only, etc)
  - May be able to detect or enumerate list of eligible profiles for a given
    cluster (eg: it looks like you have a cloud provider configured, it looks
-   like you have a storage driver, etc)
+   like you have a storage driver, etc).
  - Need to be able to selectively schedule Profile tests to nodes that support
    them (eg: Windows to windows nodes, GPU to nodes that have gpus, etc.)
 
+#### Role: SIG
+
+As a SIG community overlooking the future developments of Kubernetes,
+we must be able to identify the addition of new features under our
+ownership, and collaborate with SIG Architecture on finding the most suitable
+profile it belongs under.
+
+#### Role: Vendor
+
+As a vendor under the Certified Kubernetes program, I must identify the profiles
+supported by my distribution(s) and communicate the results to end users and
+CNCF. These results are verifiable through running profile specific conformance
+tests on a given cluster.
+
+#### Role: CNCF Conformance Program
+
+As a reviewer part of the CNCF Conformance Program, I must be able to evaluate a
+vendor submission for key criteria:
+
+- Must confirm that the version of the tests being run matches the version being
+  certified
+- Must confirm the set of tests being run matches the set of tests for the
+  version (+ profile(s)) being certified
+- Must confirm that all behaviors are covered by a test that executes, and that
+  no tests fail
```

Comment:
I'm less certain whether behaviors need to bubble up to the level of the CNCF Conformance Program.  I think it's up to the Kubernetes project itself to ensure the set of tests for a profile cover all the behaviors for a profile.  It's up to the CNCF to verify that all tests passed.

Reply:
I agree that behaviors don't need to bubble up, but it'll probably be inadvertently included since a profile consists of a set of behaviors which map to the tests. To verify that tests belong to a profile, including information about behaviors seems unavoidable.

Anyway, this isn't something that we need to explicitly define so removed the point on behavior coverage.